homeassistant/components/light/__init__.py

```diff
@@ -470,7 +470,11 @@ def state_attributes(self):
         """Return state attributes."""
         if not self.is_on:
             return None
+        return self._state_attributes
 
+    @property
+    def _state_attributes(self):
+        """Return state attributes, regardless of light's power state."""
```

Comment:
This also changes our entity model and requires an architectural decision.

Reply:
Sorry, but I don't entirely follow how this changes the entity model. The `state_attributes` property returns the exact same values as before, no?